### PR TITLE
Update GCP setup script instructions

### PR DESCRIPTION
**Pull request by gtsiolis**
### What does this PR do?

This will update the GCP setup script instructions to match the current settings and repository URL.

### CODE REVIEW

File: scripts/setup-google-adc.sh [hunk 1]
```diff
@@ -3,10 +3,10 @@
 #  1. `gcloud auth login <gcp-email>` and authenticate
 #  2. `gcloud auth application-default login` and authenticate
 #  3. `cat ~/.config/gcloud/application_default_credentials.json` and copy the output
-#  4. Go to https://gitpod.io/environment-variables/ and create:
```
Comment (gtsiolis) [inline scripts/setup-google-adc.sh:6]: **thought:** although this already redirects to `/settings`, it's best to use a URL that will probably be around in the long run.

**Review by gtsiolis — COMMENTED** [review]
@corneliusludmann feel free to take a look at this minor change. See also [relevant discussion](https://typefox.slack.com/archives/CGVQAPNPL/p1604677173241400?thread_ts=1604675453.238500&cid=CGVQAPNPL) (internal).

**Review by corneliusludmann — APPROVED** [review]
Thanks for taking care! :+1:

**Comment by gtsiolis** [comment]
Thanks @corneliusludmann! Let me know if it's ok to merge this as is or should we forward this to someone else.

**Comment by corneliusludmann** [comment]
Can be merged as is, @gtsiolis! :rocket:

**Comment by gtsiolis** [comment]
Let's merge this! 😅

**Pull request merged**